website/content/en/docs/faqs/_index.md

```diff
@@ -109,9 +109,19 @@ by adding an RBAC directive to generate a `config/rbac/role.yaml` with `update`
 
 Now run `make manifests` to update your `role.yaml`.
 
+## Seeing errors about `securityContext.runAsUser: Invalid value: 65532: must be in the ranges: [<NNN>, <MMM>]]`
+	
+For instance, if you are creating the `--kind=memcached` operator with `operator-sdk`. A workaround for this specific instance is to change the generated `config/manager/manager.yaml`:
+
+```diff
+spec:
+      securityContext:
+-        runAsUser: 65532
```

Comment:
```suggestion
```

Reply:
I think diff format is better than yaml here as it quickly shows what to remove and what to add in a commonly recognized format.

Reply:
I think it differs from the current layout of the docs. We try to make thee things easier for users are able to follow up the steps by copying and pasting them which would not be possible with this approach.